rss/medianamespace.cpp

```diff
@@ -32,8 +32,14 @@ void parse_media_node(xmlNode* node, Item& it)
 			parse_media_node(mnode, it);
 		}
 	} else if (node_is(node, "description", MEDIA_RSS_URI)) {
+		const std::string type = get_prop(node, "type");
 		if (it.description.empty()) {
 			it.description = get_content(node);
+			if (type == "html") {
+				it.description_type = "html";
+			} else {
+				it.description_type = "text";
```

Comment:
It looks like `description_type` is currently not being processed.
One effect of this is that newlines in the plaintext are ignored (I expect that `description` is always being interpreted as HTML).
An example youtube feed with multi-line descriptions on each item: https://www.youtube.com/feeds/videos.xml?channel_id=UCYO_jab_esuFRV4b17AJtAw

I'm currently considering 2 different solutions:
1. Convert the plaintext to html (escape all special characters and replace each newline with `<br>`).
2. Store the encoding for each description in the cache file.

I'm not yet sure which is better but currently I have a slight preference for option 1.

Reply:
> It looks like description_type is currently not being processed.

Yup, we even have a couple related issues: #1010 (MIME types), #468 (`type="text"`).

> Convert the plaintext to html (escape all special characters and replace each newline with <br>).

…and wrap the whole thing into `<pre>` to preserve the indentation. But that will probably break word-wrapping :(

> I'm not yet sure which is better but currently I have a slight preference for option 1.

I see how option 1 simplifies implementation, but it will hinder #1176. It might also interact poorly with [`reset-unread-on-update`](https://newsboat.org/releases/2.21/docs/newsboat.html#reset-unread-on-update): if the output of the converter changes, the article will be marked unread, even though the article wasn't actually updated.

Option 2 is harder to implement, but I believe it to be the correct path forward.

What are your reasons for preferring the first option?

Implementing either of those is a separate task, though, so they don't block this PR IMHO.

Reply:
I mainly preferred the first option to avoid introducing more complexity.
However, given the existing issues you mentioned, I agree that option 2 is the correct path forward..

Reply:
Okay, I'm marking this resolved, then. I don't think we need any more issues for this, but I'll go reference this thread in the existing two.